src/js/main-background.js

```diff
@@ -319,6 +320,25 @@ window.scroblrGlobal = (function () {
                 }, 5000);
             }
         }
+        
+        // Chrome 28+ notifications
+        if (chrome && getOptionStatus("notifications")) {
```

Comment:
Did chrome completely deprecate the `webkitNotifications` API? Just wondering if it's possible for the above condition and this one to both be true, resulting in 2 notifications being sent?